examples/index.json

```
@@ -630,6 +630,10 @@
             {
                 "name": "Cryptographic Operations",
                 "url": "crypto"
+            },
+            {
+                "name": "JWT Token Validation",
+                "url": "jwt-token-validation"
```

IMO, this should change as follows. Please update all the file names, folder names related to BBE, once the suggestion is approved by @daneshk.
```suggestion
                "name": "JWT Issue/Validate",
                "url": "jwt-issue-validate"
```